src/components/Drawer/Navigation/MenuItem/MenuItem.tsx

```diff
@@ -42,7 +42,11 @@ const MenuItem: React.FC<Props> = memo(
         <div css={menuIconStyle(isCurrent)}>
           <Icon
             name={iconName}
-            color={isCurrent ? `${pickTextColorFromSwatches(color, shade)}` : 'neutralBlack'}
+            color={
+              isCurrent
+                ? `${pickTextColorFromSwatches(color, shade)}`
```

Comment:
`pickTextColorFromSwatches` returns a string

Reply:
crrect this is what we want